warehouse/query-core/src/main/java/datawave/query/jexl/visitors/PushdownUnexecutableNodesVisitor.java

```diff
@@ -180,7 +181,8 @@ private void addDelays(JexlNode node, Object data) {
                 ExecutableDeterminationVisitor.STATE state = ExecutableDeterminationVisitor.getState(child, data, config, indexedFields, indexOnlyFields,
                                 nonEventFields, forFieldIndex, null, helper);
                 if (state == ExecutableDeterminationVisitor.STATE.PARTIAL) {
-                    ASTDelayedPredicate.create(child);
+                    JexlNode delayed = ASTDelayedPredicate.create(child);
+                    JexlNodes.replaceChild(node, child, delayed);
```

Comment:
Is this fixing another bug? Or was this functionality moved out by this PR? I didn't see it when I looked so assume this was another bug?

Reply:
One of the changes in this pull request was to not automatically reparent a source when a marker node is applied, letting the caller handle that operation. So this addition here maintains the initial functionality.